### PR TITLE
293 review work panel

### PR DESCRIPTION
**Pull request by topocount**
Please let me know if my stateless functional components need to be moved elsewhere, or if I need to modify this to conform to any other FE conventions

### CODE REVIEW

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 1]
```diff
@@ -1,20 +1,224 @@
...
+          }
+        />
+
+        <FormField
+          label="Feedback"
+          input={
+            <TextInput.Multiline
+              name='feedback'
+              rows={5}
+              style={{ resize: 'none', height: 'auto' }}
```
Comment (ottodevs) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: The current multiline from aragon ui has no top-padding should we add a little padding or reuse the [DescriptionInput](https://github.com/AutarkLabs/planning-suite/blob/7daf53218c0af317d7ea3cd645633a525bbf4ea9/apps/allocations/app/components/Form/Input/DescriptionInput.js#L10) component that has the padding already defined? (As seen on Allocations->NewProject)
Or better let it be that way? (no padding Top-bottom) cc @stellarmagnet for this

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 2]
```diff
@@ -1,20 +1,224 @@
...
+              onChange={this.changeField}
+              placeholder="Do you have any feedback to provide the contributor?"
+              wide
+            />
+          }
+        />
+
+        <ReviewRow>
+          <ReviewButton emphasis="negative" onClick={this.onReviewApplicationReject}>Reject</ReviewButton>
+          <ReviewButton emphasis="positive" onClick={this.onReviewApplicationAccept}>Accept</ReviewButton>
```
Comment (ottodevs) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: These buttons need a little horizontal separation IMHO

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 3]
```diff
@@ -1,20 +1,224 @@
...
+  line-height: 38px;
+`
+const IssueLinkRow = styled.div`
+  height: 31px;
+  display: flex;
+  align-items: center;
+  margin-bottom: 10px;
+`
+const SubmissionDetails = styled.div`
+  border: 1px solid #DAEAEF;
```
Comment (ottodevs) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Should we use theme values here? or maybe these colors are not present on the aragon ui theme

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 4]
```diff
@@ -1,20 +1,224 @@
...
+
+function CommentsCheck(props) {
+  if (!props.comments) return null
+  else{
+    return <FieldTitle>Additional Comments</FieldTitle>
+  }
 }
 
+function RatingAlert(props) {
+  return props.alert ? <Info.Alert>Please Select A Quality Rating</Info.Alert> : null
```
Comment (ottodevs) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Is it needed to have Title Case Here?

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 5]
```diff
@@ -1,20 +1,224 @@
...
+}
+
+const IssueTitle = styled(Text)`
+  color: #717171;
```
Comment (ottodevs) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Should we use theme values here? or maybe these colors are not present on the aragon ui theme

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 6]
```diff
@@ -1,20 +1,224 @@
...
+`
+const IssueLinkRow = styled.div`
+  height: 31px;
+  display: flex;
+  align-items: center;
+  margin-bottom: 10px;
+`
+const SubmissionDetails = styled.div`
+  border: 1px solid #DAEAEF;
+  background-color: #F3F9FB;
```
Comment (ottodevs) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Should we use theme values here? or maybe these colors are not present on the aragon ui theme

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 7]
```diff
@@ -1,20 +1,224 @@
...
+  margin-bottom: 14px;
+`
+const UserLink = styled.div`
+  display: flex;
+  align-items: center;
+`
+const Separator = styled.hr`
+  height: 1px;
+  width: 100%;
+  color: #D1D1D1;
```
Comment (ottodevs) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Should we use theme values here? or maybe these colors are not present on the aragon ui theme

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 8]
```diff
@@ -1,20 +1,224 @@
+
```
Comment (ottodevs) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: This empty line shouldn't be here

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js [hunk 9]
```diff
@@ -1,20 +1,224 @@
...
+  }
+
+  generateRatingChange = () => index => {
+    this.setState({rating: index})
+    console.log('index: ', index)
+  }
+
+  render() {
+    const { issue } = this.props
+    const submittor = work.user
```
Comment (ottodevs) [inline apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Is submittor an acceptable word? I am leaning more towards submitter but this is just an opinion

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js (hunk 1 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Yes, I noticed that yesterday actually, updated the other panels like this:

https://github.com/AutarkLabs/planning-suite/tree/414-fe-tweaks

File: apps/projects/app/components/Panel/ReviewWork/ReviewWork.js (hunk 2 above)
Comment (topocount) [reply apps/projects/app/components/Panel/ReviewWork/ReviewWork.js]: Changed the styled button width from a raw pixel value to a percentage, so they should be separated in any view context

**Pull request merged**